### ISSUE TITLE
[Artifact][Appareance][PvP] 

### ISSUE THREAD

**Issue by Hemoglobine-Valhalla** [Valhalla-Gaming/Tracker#7498]
**Describe the issue you're having**:
When you reach level 1 prestige and level 50 of the honor tree, you should get a quest asking you to meet king anduin in stormwind, since stormwind is phased in the past, you can't complete such quest and get your lovely appareance unlocked
**Explain how you expect it work**:
War-Torn PvP
Tint 1
Requirement: Earn the achievement The Prestige(H)/ The Prestige(A)

Acquisition: This requires that you Prestige once (in the new Legion PvP system) and reach Rank 50 again by participating in PvP activities.

Difficulty: Easy, since you can choose what activities you want to participate in, you can select easier ones such as farming battlegrounds. Requires a medium time commitment. Season 1 opens September 20th.
**Steps to reproduce the problem**:

1.  Reach prestige rank 2
2.  Try to complete the quest
3.  Notice King Anduin isn't there.

**Links to Wowhead, YouTube, etc**:

http://www.wowhead.com/quest=42978/a-royal-audience#comments

**Comment by aetoni** [comment]
I am having the same problem here and i worked hard to get there, should i just proceed to prestige lvl 2?

**Comment by Hemoglobine-Valhalla** [comment]
Yeah, just keep either the quest or the item and don't delete it.

**Comment by Traveler-Valhalla** [comment]
it's same for horde version
http://www.wowhead.com/quest=42985/a-royal-audience
